### PR TITLE
Add new Red Ball series autosplitter

### PR DESCRIPTION
**Pull request by ItsMaximum**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -15566,32 +15566,34 @@
...
-            <Game>Red and Blue Balls 3</Game>
+            <Game>Multiple Red Ball Games</Game>
+            <Game>Red Ball Category Extensions</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/RedBallGames.asl</URL>
+            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/rbFullSeries.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Splits, starts, and resets. Click "Website" for the right game version and the README. (By Maximum, Ero)</Description>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:15594]: What's the reasoning for removing an author here?

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:15594]: I do not care.

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -15566,32 +15566,34 @@
...
+            <Game>Red and Blue Balls 2</Game>
+            <Game>Red and Blue Balls 3</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/rb1improved.asl</URL>
+            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/RedBallGames.asl</URL>
         </URLs>
         <Type>Script</Type>
         <Description>Perfectly splits upon completing each level. Click "Website" to download the game. (By Maximum)</Description>
         <Website>https://drive.google.com/file/d/12X0J05rZ6gRnPgybDJuz56xGGnxjaOev/view?usp=sharing</Website>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:15579]: Is this still the correct download link, now that you've changed which game the autosplitter is for?

File: LiveSplit.AutoSplitters.xml [hunk 3]
```diff
@@ -15566,32 +15566,34 @@
...
+            <Game>Red Ball Category Extensions</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/RedBallGames.asl</URL>
+            <URL>https://raw.githubusercontent.com/ItsMaximum/autosplitters/master/rbFullSeries.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Splits, starts, and resets. Click "Website" for the right game version and the README. (By Maximum, Ero)</Description>
+        <Description>Perfectly splits upon completing each level. Click "Website" to download the games. (By Maximum)</Description>
         <Website>https://drive.google.com/file/d/1dvM7VoIdnX9sEo7OB6J7ybNcZG_ZrCpp/view?usp=sharing</Website>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:15597]: Is this still the correct link?

File: LiveSplit.AutoSplitters.xml (hunk 2 above)
Comment (ItsMaximum) [reply LiveSplit.AutoSplitters.xml:15579]: Yes, the side games still use the old autosplitter (Red Ball 5 + the Red and Blue Balls games)

File: LiveSplit.AutoSplitters.xml (hunk 3 above)
Comment (ItsMaximum) [reply LiveSplit.AutoSplitters.xml:15597]: yes

Comment (wooferzfg) [reply LiveSplit.AutoSplitters.xml:15597]: I was asking about the Google Drive link

File: LiveSplit.AutoSplitters.xml (hunk 2 above)
Comment (wooferzfg) [reply LiveSplit.AutoSplitters.xml:15579]: I was asking about the Google Drive link

**Pull request merged**